tests/store/test_footprints.py

```diff
@@ -687,50 +688,70 @@ def test_optional_metadata():
         inlet=inlet,
         species=species,
         domain=domain,
-        optional_metadata={"project":"test"},
+        optional_metadata={"project": "test"},
     )
 
     # Get the footprints data
-    footprint_results = search(site=site, domain=domain, species=species, data_type="footprints",)
+    footprint_results = search(
+        site=site,
+        domain=domain,
+        species=species,
+        data_type="footprints",
+    )
 
     footprint_obs = footprint_results.retrieve_all()
     footprint_metadata = footprint_obs.metadata
 
     assert "project" in footprint_metadata
 
 
-def test_fp_retrieved_from_species():
-    """
-    Test to verify species are converted to synonym values and retrieved
+# These tests test that custom keys can be used to create unique Datasources
+@pytest.fixture()
+def mock_metakeys():
+    # TODO - implement this in a different way
+    default_keys = get_metakey_defaults()
+
+    default_keys["footprints"]["optional"] = ["project", "special_tag"]
+
+    with patch("openghg.store.base._base.get_metakeys", return_value=default_keys):
+        yield
+
+
+def test_standardise_footprints_different_datasources(mock_metakeys):
```

Comment:
To make our own life easier later on, we should probably mark this test in some way / add a comment that we expect this functionality to change - at this moment this explicitly uses `optional_metadata` to include new distinguishing keys which we'll want to move away from as we go forward.

Reply:
I wrote down the name of the tests, I'll try to add it to the notes in the umbrella issue. This test will probably fail or be found when `optional_metadata` is renamed though.